src/main/java/com/github/packageurl/PackageURL.java

```diff
@@ -94,7 +94,8 @@ public PackageURL(final String type, final String name) throws MalformedPackageU
     public PackageURL(final String type, final String namespace, final String name, final String version,
                       final TreeMap<String, String> qualifiers, final String subpath)
             throws MalformedPackageURLException {
-        this.type = validateType(type);
+        this.scheme = validateScheme("pkg");
```

Comment:
Apparently, I failed at fixing the merge conflict. Sorry about that.
```suggestion
        this.scheme = validateScheme("pkg");
```